chapters/file-tags.md

```diff
@@ -50,10 +50,62 @@ Multiple-line values are not recommended, because doing so will make it harder f
 
 Version 3.0 of the [REUSE Software guidelines](https://reuse.software/spec/) implements this format, via a recommendation to use the tag `SPDX-FileCopyrightText:` to include copyright notices as part of a file's comment headers.
 
-## H.3 Caveats <a name="H.3"></a>
+## H.3 File Tags Within Snippets <a name="H.3"></a>
+
+If certain SPDX file tags are to apply only to a certain snippet instead of the whole file, the start of such snippet should be marked with `SPDX-SnippetBegin` and its end with `SPDX-SnippetBegin`.
```

Comment:
Ahem. Probably more useful to denote the end with `SPDX-SnippetEnd` instead with `Begin`.